File: ProjetoDeAplicativos/ProjetoDeAplicativos/ServicoClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Data;

namespace ProjetoDeAplicativos
{
    class ServicoClass
    {
        public string codServico { get; set; }
        public string nome { get; set; }
        public string descricao { get; set; }
        public string valor { get; set; }


        public DataTable retServicos()
        {
            ClasseDB db = new ClasseDB();
            return db.RetornarTabela("SELECT * FROM Servico");
        }

        public void inserirServico()
        {
            ClasseDB db = new ClasseDB();
            db.ExecutarComando($"INSERT INTO Servico (codServico, nome, descricao, valor) VALUES ('{codServico}', '{nome}', '{descricao}', '{valor}')");
        }

        public void editarServico()
        {
            ClasseDB db = new ClasseDB();
            db.ExecutarComando($"UPDATE Servico SET codServico='{codServico}', nome='{nome}',descricao='{descricao}', valor='{valor}' WHERE codServico='{codServico}'");
        }

        public void excluirServico()
        {
            ClasseDB db = new ClasseDB();
            db.ExecutarComando($"DELETE FROM Servico WHERE codServico='{codServico}'");
        }

        public string retProxCodServico()
        {
            DataTable tableservicos = retServicos();
            int max = tableservicos.Rows.Count;
            string codAntigo = tableservicos.Rows[max - 1]["codServico"].ToString();
            codAntigo = codAntigo.Substring(4, 3);
            int numNovo = int.Parse(codAntigo) + 1;
            string numString = numNovo.ToString();
            while (numString.Length < 3)
            {
                numString = "0" + numString;
            }
            string codNovo = "SER" + numString;
            return codNovo;
        }
    }
}

File: http/class/ServicoClass.php
<?php

class ServicoClass {

    //-- Propriedades privadas
    private $codServico;
    private $nome;
    private $descricao;
    private $valor;    

    //-- Encapsulamento das propriedades (Método GET e SET)
    function getCodServico() {
        return $this->codServico;
    }

    function getNome() {
        return $this->nome;
    }

    function getDescricao() {
        return $this->descricao;
    }

    function getValor() {
        return $this->valor;
    }


    function setCodServico($codServico) {
        $this->codServico = $codServico;
    }

    function setNome($nome) {
        $this->nome = $nome;
    }

    function setDescricao($descricao) {
        $this->descricao = $descricao;
    }

    function setValor($valor) {
        $this->valor = $valor;
    }

            //-------------- CRUD-----------------------
    public function retServicos() {
        require_once('class/ConexaoClass.php');
        $objConexao = new ConexaoClass("localhost", "root", "", "dbPrimoPet");
        $tableServicos = $objConexao->selecionarDados("SELECT * FROM Servico");

        return $tableServicos;
    }
    
    public function inserirServico($objServico) {
        require_once('class/ConexaoClass.php');
        $objConexao = new ConexaoClass("localhost", "root", "", "dbPrimoPet");   
        
        $codServico = $objServico->getCodServico();
        $nome = $objServico->getNome();
        $descricao = $objServico->getDescricao();
        $valor = $objServico->getValor();
                       
        $objConexao->exercutarComandoSQL("INSERT INTO Servico (codServico, nome, descricao, valor) VALUES ('$codServico', '$nome', '$descricao', '$valor')");               

        return true;
    }
    
     public function editarServico($objServico) {
        require_once('class/ConexaoClass.php');
        $objConexao = new ConexaoClass("localhost", "root", "", "dbPrimoPet");   
        
        $codServico = $objServico->getCodServico();
        $nome = $objServico->getNome();
        $descricao = $objServico->getDescricao();
        $valor = $objServico->getValor();
       
        $objConexao->exercutarComandoSQL("UPDATE Servico SET codServico='$codServico', nome='$nome',descricao='$descricao', valor='$valor' WHERE codServico='$codServico'");               

        return true;
    }
    
    public function excluirServico($codServico) {
        require_once('class/ConexaoClass.php');
        $objConexao = new ConexaoClass("localhost", "root", "", "dbPrimoPet");   
        
        $objConexao->exercutarComandoSQL("DELETE FROM Servico WHERE codServico='$codServico'");               

        return true;
    }
    
    public function retProxCodServico(){
        $tableclientes = $this->retServicos();
        if ($tableclientes == "") {
            $codNovo = 'SER000';
        }
        else{
        $max = sizeof($tableclientes);
        $codAntigo  = $tableclientes[$max-1]['codServico'];
        $codAntigo = substr($codAntigo,4,3);
        $numNovo =  intval($codAntigo)+1;
        $numString = (string)$numNovo;
        while (strlen($numString) < 3){
            $numString = '0'.$numString;
        }
        $codNovo = 'SER'.$numString;
        }
        return $codNovo;
    }
}